odoo/tools/translate.py

```diff
@@ -199,7 +208,8 @@ def nonspace(text):
     def translatable(node):
         """ Return whether the given node can be translated as a whole. """
         return (
-            node.tag in TRANSLATED_ELEMENTS
+            should_force_translatable_element(node)
```

Comment:
Maybe just `"o_force_translatable" in node.attrib.get("class", "").split()`

Reply:
I mean why you need an independent function for `should_force_translatable_element`?